_data/simple-icons.json

```
@@ -775,6 +775,11 @@
             "hex": "0F2346",
             "source": "https://www.appannie.com/en/about/press/"
         },
+        {
+            "title": "App Center",
+            "hex": "CB2E62",
+            "source": "https://visualstudio.microsoft.com/app-center/"
```

```suggestion
            "source": "https://learn.microsoft.com/en-us/visualstudio/subscriptions"
```

SVG version of the logo available here.